src/server/frontend_xwayland/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+set(
+  XWAYLAND_SOURCES
+
+  xwayland_default_configuration.cpp
+  xwayland_connector.cpp  xwayland_connector.h
+  xwayland_server.cpp     xwayland_server.h
+  xwayland_wm.cpp         xwayland_wm.h
+  xwayland_wm_surface.cpp xwayland_wm_surface.h
+  xwayland_wm_shellsurface.cpp xwayland_wm_shellsurface.h
+  xwayland_wm_shell.cpp xwayland_wm_shell.h
+)
+
+include_directories(../frontend_wayland)
```

Comment:
This too make it look as though this is a part of frontend_wayland, not a separate implementation.

Reply:
This is only to "Hook" into the wayland variables. I use that to create the shell inside the wayland connector then expose it, instead of exposing the vars raw.